english/check-phone/check-phone.md

```diff
@@ -0,0 +1,7 @@
+You get a notification that someone has texted you, but you don't 
+recognize the number. You open your messaging app, but you can't see
+a preview of the text. Do you:
+
+[Open the message](../open-message.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.